integration/spark2/src/main/scala/org/apache/spark/sql/execution/command/management/CarbonLoadDataCommand.scala

```diff
@@ -445,13 +447,11 @@ case class CarbonLoadDataCommand(
       case c: CatalogRelation => c
     }.head.asInstanceOf[LogicalPlan]
 
+
```

Comment:
comment for line 437 (function signature):
1. suggest to rename `loadDataWithPartition`
2. from signature, user can not easily tell why it is for partition, there is no parameter is partition related. It is similar to `CarbonDataRDDFactory.loadCarbonData` only, can we make it more readable.
3. there are many repeated code in if block and else block, can we extract common part to anther function

Reply:
1.Ok, I have changed the method name. 
2.It is just used InsertIntoCommand to let spark handles the partitioning so it does not require to know anything about partitioning information.
3. I don't see any common code to extract it, both if else has different logic.
4. Added comment